File: Data Structures & Algorithms/two-integer-sum/submission-12.go
func twoSum(nums []int, target int) []int {
    hashmap:=make(map[int]int)
    for i,value:=range nums{
        key:=target-value
        if comp,ok:=hashmap[key];ok{
            return[]int{comp,i}
        }else{
            hashmap[value]=i
        }
    }
    return nil
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-12.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        hashmap={}
        count=[[]for i in range(len(nums)+1)]
        for num in nums:
            hashmap[num]=hashmap.get(num,0)+1
        for num,freq in hashmap.items():
            count[freq].append(num)
        res=[]
        for i in range(len(count)-1,0,-1):
                for num in count[i]:
                 res.append(num)
                 if k==len(res):
                  return res
            
        
